### ISSUE TITLE
Encode creator/curator into XML

### ISSUE THREAD

**Issue by jonrkarr** [sys-bio/libOmexMeta#85]
Creators and curators are encoded identically into XML. Creator vs. curator needs to be encoded into XML for this information to be preserved through export/import to/from XML.

**Comment by jonrkarr** [comment]
For curators, what about using the MARC editor refinement for dc:contributor?
http://www.loc.gov/loc.terms/relators/EDT
http://memory.loc.gov/diglib/loc.terms/relators/dc-contributor.html
https://www.dublincore.org/specifications/dublin-core/usageguide/appendix_roles/

```xml
<rdf:Description rdf:about="http://www.loc.gov/loc.terms/relators/EDT">
    <rdfs:subPropertyOf rdf:resource="http://purl.org/dc/elements/1.1/contributor" />
</rdf:Description>
```

SPAR has also given deeper consideration to this:
https://sparontologies.github.io/pro/current/pro.html#namedindividuals
https://sparontologies.github.io/scoro/current/scoro.html#namedindividuals

**Comment by nickerso** [comment]
Currently the OMEX Metadata Spec defines the model author/creator as the `creator` of the model itself (e.g., SBML or CellML model encoding), and the curator as the `creator` of the annotation file. This was an initial attempt at representing this knowledge in a way that is consistent with current usage. For future versions of the OMEX Metadata Spec, I'm definitely keen to see us explore capturing richer provenance knowledge which begins to allow the distinction of types of contrinutions (i.e., model author vs person who encoded model into SBML vs tool that was used to encode the model...).

**Comment by jonrkarr** [comment]
That makes sense. In many cases, authors (i.e. `creator` of the model) will provide at least some of the information that goes into an OMEX Meta file. Under the pyomexmeta scheme, does that mean that authors will also be described as `creators` of the OMEX Meta files, in which case pyometa meta will interpret these people to also be "curators"?

**Comment by nickerso** [comment]
yep, in the current scheme that is how this would work. Hopefully since 1.1 metadata spec has focussed on resolving issues with representing biological knowledge, we'll then be able to better focus on non-biology semantics (such as provenace) in the next round of updates. And then as the spec defines how to represent this knowledge, pyomexmeta would be updated to support it.

ping @jhgennari and @maxneal for awareness of this discussion in case I'm getting things wrong :)
